src/markdown-pages/install-with-kurl/preflights.md

```diff
@@ -27,6 +27,7 @@ The following checks run on all nodes where kURL is installed:
 * TCP port 10259 is available for the kube scheduler.
 * At least 4 GiB of memory is available. (Warn when less than 8GiB).
 * /var/lib/kubelet has at least 30GiB total space and is less than 80% full. (Warn when more than 60% full).
+* /var/lib/docker has at least 30GiB total space and is less than 80% full. (Warn when more than 60% full).
```

Comment:
Isn't this only run on installations with docker?

Can we mention that?

Reply:
That's a good point. Maybe add a section for Docker at the bottom like Rook has and put it there.

Reply:
Moved the docs under the add-on section. I think that implies it only runs if the docker add-on is enabled.